draft-ietf-core-yang-cbor.md

```diff
@@ -261,6 +261,22 @@ by the environment.
 
 Mechanisms and processes used to assign SIDs to YANG items and to guarantee their uniqueness are outside the scope of the present specification.
 If SIDs are to be used, the present specification is used in conjunction with a specification defining this management.
+The objectives for such a management scheme need to include:
+
+* Keep the mapping from YANG SIDs to YANG names ({{name}}) a mathematical
+  function — a single YANG SID is only ever associated with a
+  single YANG name.
+* Keep the name-SID mapping "mostly" a function — while many
+  management processes cannot entirely prevent ever making several
+  independent allocations of a YANG SID for a YANG name, one objective
+  would be to have a single “canonical” SID mapping for each YANG
+  name, preferably associated with the same provenance that the YANG
+  module defining the YANG name itself has.
+
```

Comment:
I don't know what this paragraph means, or see any use-cases, so don't agree this paragraph should be included

Reply:
This paragraph was part of what we discussed in IETF113, so I would like to keep it.
The point is that you can write a tool that uses the canonical name-SID mapping and there is a very good chance that this will interoperate.

Reply:
It isn't up to me what text is included, so leave it in.
I don't know what this means for SID files and server implementations.
A SID is permanent, except when it is temporary.
A permanent SID to data node mapping is permanent, except when it gets reused for something else.
IMO a lot of unproven requirements added at the end.

I will not be relying on global SID files so ad-hoc won't really change anything,
but those who are should have some concern. Diagnostics support is needed the most
at the start, not something to add years later.